File: html5apps/terminal/events/core/wubook/WubookBooking.php
<?php
class core_wubook_WubookBooking {
	/** @var String */
	public $channelId;

	/** @var String */
	public $reservationCode;

	/** @var String */
	public $channel_reservation_code;

	/** @var String */
	public $delete;

	/** @var String */
	public $customerNotes;

	/** @var String */
	public $phone;

	/** @var String */
	public $wasModified;

	/** @var String */
	public $email;

	/** @var String */
	public $city;

	/** @var String */
	public $postCode;

	/** @var String */
	public $address;

	/** @var String */
	public $name;

	/** @var String */
	public $depDate;

	/** @var String */
	public $arrivalDate;

	/** @var core_wubook_WubookBookedRoom[] */
	public $rooms;

	/** @var String */
	public $modifiedReservation;

	/** @var String */
	public $status;

	/** @var String */
	public $countryCode;

	/** @var String */
	public $isExpediaCollect;

	/** @var String */
	public $isNonRefundable;

	/** @var String */
	public $isBookingComVirtual;

	/** @var String */
	public $isPrePaid;

	/** @var String */
	public $isAddedToPms;

}
?>

File: messages/src/main/java/com/thundashop/core/wubook/WubookBooking.java

package com.thundashop.core.wubook;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class WubookBooking implements Serializable {
    public String channelId;
    public String reservationCode;
    public String channel_reservation_code;
    public boolean delete;
    public String customerNotes;
    public String phone;
    public Integer wasModified;
    public String email;
    public String city;
    public String postCode;
    public String address;
    public String name;
    public Date depDate;
    public Date arrivalDate;
    public List<WubookBookedRoom> rooms = new ArrayList();
    public List<Integer> modifiedReservation = new ArrayList();
    public Integer status;
    public String countryCode;
    public boolean isExpediaCollect = false;
    boolean isNonRefundable = false;
    boolean isBookingComVirtual = false;
    boolean isPrePaid = false;
    
    boolean isAddedToPms = false;
    public String language = "";
}
